cluster-configuration/services-configuration.yaml

```diff
@@ -69,9 +69,35 @@ hadoop:
       capacity: 20
 
 
+hadoop-name-node: 
+  # which role of machine to deploy on
+  role: pai-master
+
+
+hadoop-resource-manager:
+  # which role of machine to deploy on 
+  role: pai-master
+
+
+hadoop-data-node: 
+  # which role of machine to deploy on
+  role: pai-worker
+
+
+hadoop-node-manager: 
+  # which role of machine to deploy on
+  role: pai-worker
+
+
+hadoop-jobhistory: 
+  # which role of machine to deploy on
+  role: pai-master
+
 
 frameworklauncher:
   frameworklauncher-port: 9086
+  # which role of machine to deploy on
+  role: pai-master
 
```

Comment:
Information put here centrally will put under each service directory respectively.

Reply:
Have put the config under each service's dir